odoo/addons/base/ir/ir_cron.py

```diff
@@ -218,8 +226,21 @@ def _acquire_job(cls, db_name):
                 # we're exiting due to an exception while acquiring the lock
                 lock_cr.close()
 
+        try:
+            with db.cursor() as cr:
+                cr.execute("SELECT MIN(nextcall) as nextcall FROM ir_cron WHERE active = 't'")
+                row = cr.dictfetchone()
+                if row:
+                    res['nextcall'] = row['nextcall']
+                    min_cron = datetime.strptime(row['nextcall'], DEFAULT_SERVER_DATETIME_FORMAT)
+                    epoch = datetime.utcfromtimestamp(0)
+                    res['nextcall_epoch'] = int((min_cron - epoch).total_seconds())
```

Comment:
Seems weird that `_acquire_job` would return *two* "nextcall" values that the standard server does not use, as if one was not enough ;-) Do we really need both? Maybe we could just return `row['nextcall']` and the epoch conversion could be done by the caller?

By the way, since we know we're dealing with UTC, the conversion might be simplified further:
```python
  nextcall = fields.Datetime.from_string(row['nextcall'])
  res['nextcall_epoch'] = time.mktime(nextcall.timetuple())
```